portal-impl/test/unit/com/liferay/portal/dao/sql/transformer/BaseSQLTransformerLogicTestCase.java

```diff
@@ -164,6 +171,14 @@ public void testTransform() {
 		Assert.assertEquals(sql, sqlTransformer.transform(sql));
 	}
 
+	protected String getAggregationOriginalSQL() {
+		return "select foo from Foo order by AGGREGATION_BOOLEAN_MIN(foo)";
+	}
+
+	protected String getAggregationTransformedSQL() {
+		return "select foo from Foo order by AGGREGATION_BOOLEAN_MIN(foo)";
```

Comment:
Should the transformed SQL be like:

`select foo from Foo order by min(foo)`

Reply:
Fixed and push forced.